bash_completion

```diff
@@ -3011,21 +3011,19 @@ __load_completion()
                     # At least $cmd is expected to have a completion set when
                     # we return successfully; see if it already does
                     if compspec=$(complete -p "$cmd" 2>/dev/null); then
-                        local -a extspecs=()
                         # $cmd is the case in which we do backslash processing
-                        [[ $backslash ]] && extspecs+=("$backslash$cmd")
+                        [[ $backslash ]] && eval "$compspec \"$backslash$cmd\""
```

Comment:
```suggestion
                        [[ $backslash ]] && eval "$compspec \"\$backslash\$cmd\""
```

To avoid double expansions (for the case where e.g., the value of `cmd` contains `"` or `$(...)`).